src/fidesops/api/v1/endpoints/user_endpoints.py

```diff
@@ -89,6 +96,75 @@ def create_user(
     return user
 
 
+def _validate_current_user(user_id: str, user_from_token: FidesopsUser) -> bool:
+    if not user_from_token:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
+            detail=f"User with ID {user_id} does not exist.",
+        )
+
+    if user_id != user_from_token.id:
+        raise HTTPException(
+            status_code=HTTP_401_UNAUTHORIZED,
+            detail=f"You are only authorised to update your own user data.",
```

Comment:
Just checking here (sorry if I'm reading this wrong!), but I think the idea is that if the user has those admin privileges, then they can update a user's data as well, not just that user themselves. @adriaaaa can you confirm?

Reply:
Which admin privs specifically @LKCSmith ? If it's the usercreate and userupdate then yes that user should have the ability to create/update other users.

Reply:
@adriaaaa From the mocks, it looks an admin can update a user's name, password, and privileges?

Reply:
Change made 👍🏻